### ISSUE TITLE
Update -> 0.31.0

### ISSUE THREAD

**Issue by ghost** [h3rald/vscode-min-lang#1]
The extension lacks v0.31.0 *min*, extension should be updated to version v0.31.0.

**Comment by h3rald** [comment]
Yes yes I know! Gotta remove some highlighting for some of the old sigils, add `lambda` etc.

I actually do most of my min coding on my iPhone and iPad these days, while for the extension I need to try it out on a laptop, so... 😬

**Comment by ghost** [comment]
Sorry 😁
